tests/test_compiler.py

```diff
@@ -53,7 +53,7 @@ def circuit():
         return qml.state()
 
     with pytest.raises(
-        CompileError, match="PennyLane-Catalyst 0.5.0 or greater is required, but installed 0.0.1"
+        CompileError, match="PennyLane-Catalyst 0.6.0 or greater is required, but installed 0.0.1"
```

Comment:
Not sure if regex or something is allowed here? Alternatively I usually just truncate the matched error message, but here the version is in the middle 🤔